### PR TITLE
Update tech-blog.md

### PR DESCRIPTION
**Pull request by terrikwak**
Added new blog articles

### CODE REVIEW

File: tech-blog.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ Here you can find our latest articles and updates from the [Klaytn Tech Blog](ht
...
+
+How to run a local Klaytn network ([한국어](https://medium.com/klaytn-kr/%ED%81%B4%EB%A0%88%EC%9D%B4%ED%8A%BC-%EB%A1%9C%EC%BB%AC-%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0-229c75e940f9), [English](https://medium.com/klaytn/how-to-run-a-local-klaytn-network-9b7a63da04f2) )
+
+Using Klaytn Plugin on Remix ([한국어](https://medium.com/klaytn-kr/%EB%A6%AC%EB%AF%B9%EC%8A%A4%EC%97%90%EC%84%9C-%ED%81%B4%EB%A0%88%EC%9D%B4%ED%8A%BC-%ED%94%8C%EB%9F%AC%EA%B7%B8%EC%9D%B8-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0-3394b52cc8fd), [English](https://medium.com/klaytn/using-klaytn-plugin-on-remix-320ead9114fe) )
+
+Using SnapShot for your Klaytn Project Governance ([한국어](https://medium.com/klaytn-kr/%ED%81%B4%EB%A0%88%EC%9D%B4%ED%8A%BC-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%EA%B1%B0%EB%B2%84%EB%84%8C%EC%8A%A4%EB%A5%BC-%EC%9C%84%ED%95%9C-%EC%8A%A4%EB%83%85%EC%83%B7-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0-c31f27a9fecb), [English](https://medium.com/klaytn/using-snapshot-for-your-klaytn-project-governance-1ced3ff6244) )
```
Comment (kjhman21) [inline tech-blog.md:18]: @terrikwak Can you also add tx latency blog?

Comment (kjhman21) [reply tech-blog.md:18]: https://medium.com/klaytn/a-comparison-of-blockchain-network-latencies-7508509b8460

Comment (terrikwak) [inline tech-blog.md:18]:
```suggestion
Using SnapShot for your Klaytn Project Governance ([한국어](https://medium.com/klaytn-kr/%ED%81%B4%EB%A0%88%EC%9D%B4%ED%8A%BC-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%EA%B1%B0%EB%B2%84%EB%84%8C%EC%8A%A4%EB%A5%BC-%EC%9C%84%ED%95%9C-%EC%8A%A4%EB%83%85%EC%83%B7-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0-c31f27a9fecb), [English](https://medium.com/klaytn/using-snapshot-for-your-klaytn-project-governance-1ced3ff6244) )

A Comparison of blockchain network latencies ([한국어](https://medium.com/klaytn-kr/%EB%A9%94%ED%83%80%EB%B2%84%EC%8A%A4%EC%97%90-%EC%A0%81%ED%95%A9%ED%95%9C-%EB%B8%94%EB%A1%9D%EC%B2%B4%EC%9D%B8%EC%9D%80-d42b7a437616), [English](https://medium.com/klaytn/a-comparison-of-blockchain-network-latencies-7508509b8460) )

```

File: tech-blog.md [hunk 2]
```diff
@@ -9,6 +9,17 @@ Here you can find our latest articles and updates from the [Klaytn Tech Blog](ht
...
+
+A Comparison of blockchain network latencies ([한국어](https://medium.com/klaytn-kr/%EB%A9%94%ED%83%80%EB%B2%84%EC%8A%A4%EC%97%90-%EC%A0%81%ED%95%A9%ED%95%9C-%EB%B8%94%EB%A1%9D%EC%B2%B4%EC%9D%B8%EC%9D%80-d42b7a437616), [English](https://medium.com/klaytn/a-comparison-of-blockchain-network-latencies-7508509b8460) )
+
```
Comment (terrikwak) [inline tech-blog.md:21]:
```suggestion
```

**Review by kjhman21 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**